features/step_definitions/files_steps.rb

```diff
@@ -12,8 +12,7 @@
 
 
 Then(/^(?:now I|I still) have the following committed files$/) do |files_data|
-  files_data.map_headers!(&:downcase)
-  verify_files files_data.hashes
+  files_data.diff! files_in_branches
```

Comment:
The way I understand this, we are using the headers of the Cucumber table as the headers of our own table. Therefore, if somebody writes FLIE instead of FILE, it would pass when it shouldn't, no?